docs/source/overview/environments.rst

```diff
@@ -53,8 +53,10 @@ Classic environments that are based on IsaacGymEnvs implementation of MuJoCo-sty
     |                  |                             |                                                                         |
     |                  | |cartpole-direct-link|      |                                                                         |
     +------------------+-----------------------------+-------------------------------------------------------------------------+
-    | |cartpole|       | |cartpole-camera-rgb-link|  | Move the cart to keep the pole upwards in the classic cartpole control  |
-    |                  |                             | using perceptive inputs                                                 |
+    | |cartpole|       | |manager-camera-rgb-link|   | Move the cart to keep the pole upwards in the classic cartpole control  |
+    |                  |                             | and perceptive inputs                                                   |
+    |                  | |manager-camera-dpt-link|   |                                                                         |
+    |                  | |cartpole-camera-rgb-link|  |                                                                         |
```

Comment:
Can you add blank lines between the various links?
This is a new addition we made in 1.2 to make the table look nicer